zaza/model.py

```diff
@@ -90,20 +90,26 @@ def get_juju_model():
         await model.disconnect()
 
 
-def get_unit_from_name(unit_name, model):
+def get_unit_from_name(unit_name, model=None, model_name=None):
     """Return the units that corresponds to the name in the given model.
 
     :param unit_name: Name of unit to match
     :type unit_name: str
     :param model: Model to perform lookup in
-    :type model: juju.model.Model
+    :type model: model.Model()
+    :param model_name: Name of the model to perform lookup in
+    :type model_name: string or model.Model()
```

Comment:
nit: this is probably left over from the initial proposal

Reply:
Thanks!